test/controllers/users_controller_test.rb

```diff
@@ -32,4 +32,41 @@
       expect(flash[:result_text]).must_equal "User not found!"
     end
   end
+
+  describe "login" do
+    it "can log in an existing user" do
+      # Arrange
+      user_count = User.count
+
+      # Act
+      user = perform_login
+
+      expect(user_count).must_equal User.count
```

Comment:
is this actually testing anything? or rather, i'm not sure what this is testing here

Reply:
i might have an expect here, like: expect { user = perform_login(user) }.wont_change "User.count"